gossip/src/stake_weighting_config.rs

```diff
@@ -0,0 +1,31 @@
+use {
+    serde::{Deserialize, Serialize},
+    solana_account::ReadableAccount,
+    solana_runtime::bank::Bank,
+};
+
+#[derive(Serialize, Deserialize, Debug, Default, Clone)]
+#[repr(C)]
+pub struct WeightingConfig {
```

Comment:
I'm not sure but would you not get some additional header written here by record program? Or are you using a different program to write onchain state?

Reply:
It may also be nice to comment why we are deriving Serialize here as we are only deserializing this one.